.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
If this is not directly related to the streaming API, please submit changes into a pull request in the main repo.